physionet-django/user/views.py

```diff
@@ -286,6 +296,17 @@ def verify_email(request, uidb64, token):
             associated_email.verification_date = timezone.now()
             associated_email.is_verified = True
             associated_email.save()
+
```

Comment:
Can you avoid this duplication with something like a `legacycredentialed(email)` function? Might be nice to separate it out.